Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/PDFDataDetectorOverlayController.mm

```diff
@@ -128,20 +132,25 @@
     if (!plugin)
         return false;
 
+    RefPtr mainFrameView = plugin->mainFrameView();
+    if (!mainFrameView)
+        return false;
+
     if (!PAL::isDataDetectorsFrameworkAvailable())
         return false;
 
     updateDataDetectorHighlightsIfNeeded(pageIndex);
 
-    auto mousePositionInPluginSpace = plugin->mousePositionInView(event);
+    auto mousePositionInWindowSpace = event.position();
```

Comment:
nit: What do you think about renaming this to `mousePositionInRootView`, since that's the nomenclature we've been rolling with in UnifiedPDF?

Reply:
(repeat for other `mousePositionInWindowSpace` variables in this PR)

Reply:
I think that might be a little strange, since the variable is passed into `windowToContents`.

Reply:
Hmm, yes, `mousePositionInWindowSpace` reads better given how we're using it. It might seem like "yet another coordinate space" for an uninitiated reader, although I don't know how valid that claim is.

Reply:
(Feel free to ignore!)